HymnsTests/Display/DisplayHymnViewModelSpec.swift

```diff
@@ -36,20 +37,8 @@ class DisplayHymnViewModelSpec: QuickSpec {
                         it("title should be empty") {
                             expect(target.title).to(beEmpty())
                         }
-                        it("lyrics should be nil") {
-                            expect(target.lyrics).to(beNil())
-                        }
-                        it("chords should be nil") {
-                            expect(target.chordsUrl).to(beNil())
-                        }
-                        it("guitar url should be nil") {
-                            expect(target.guitarUrl).to(beNil())
-                        }
-                        it("piano url should be nil") {
-                            expect(target.pianoUrl).to(beNil())
-                        }
```

Comment:
I don't know if you're just not done with the tests yet, but after deleting these cases, it'd probably be better to add new cases testing the new behavior.

Reply:
Added a new test verifying the number of tabs. Is there any other functionality or things you think we should test?

Reply:
I don't think we made that big of a change to the view model, so there's not that much to change.

But I do think there's some value in testing the ordering of the tabs, since that affects the order the tabs appear on the screen. Could you add those to the "should have x tabs" assertions too?

You can use the id to perform the assertion. So like 

```
expect(target.tabItems[0].id).to(equal("Lyrics"))
expect(target.tabItems[1].id).to(equal("Chrods"))
```
etc

Other than that, everything else looks good!